### ISSUE TITLE
Doesn't work..

### ISSUE THREAD

**Issue by Suleman-Elahi** [marchenko1985/ExcelPostJsonAddIn#1]
I tried it on Windows 10 and I don't think that it works.

**Comment by marchenko1985** [comment]
Yep, it can be not working because of outdated everything, demo was made way too long ago, by not working you mean that project is not compiling or compiled plugin not installing or backend demo not catching request?

**Comment by Suleman-Elahi** [comment]
The plugin is installed but it can't make the requests. It says enable JavaScript in your browser, throws an error and closes.....
